package/rubygem-packaging_rake_tasks.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Thu May 14 12:06:01 UTC 2015 - jreidinger@suse.com
+
+- Do not crash if tarball have different name in devel project
```

Comment:
same as before. It is fix of previous change and needed mainly in jenkins. So not critical